箇条書きが全角のコロンで区切られている場合に検知されない

## 要旨

`- **重要情報**：これは重要な項目です`のように全角のコロンで強調部分が区切られている場合に検知されない。

半角コロン`:`で区切られる場合と文章の意味は同じであると見なせるので、検知できるようにしたい。

## 再現手順

```sh
$ npm install textlint@15.2.1 @textlint-ja/textlint-rule-preset-ai-writing@1.6.0
$ echo '{ "rules": { "@textlint-ja/preset-ai-writing": true } }' > .textlintrc.json
$ echo "- **重要情報**：これは重要な項目です" > sample.md
$ echo "- **重要情報**: これは重要な項目です" >> sample.md
$ npx textlint sample.md
```

半角コロン`:`で区切られているもののみ検出されます。

```console
/private/tmp/sandbox/reproduction/sample.md
  2:1  error  リストアイテムで強調（**）とコロン（:）の組み合わせは機械的な印象を与える可能性があります。より自然な表現を検討してください。  @textlint-ja/ai-writing/no-ai-list-formatting

✖ 1 problem (1 error, 0 warnings, 0 infos)
```

## 環境

- OS: `macOS Sequoia 15.6 arm64`
- node: `v22.18.0`
- textlint: `15.2.1`
- `@textlint-ja/textlint-rule-preset-ai-writing`: `1.6.0`

--- a/src/rules/no-ai-list-formatting.ts
+++ b/src/rules/no-ai-list-formatting.ts
@@ -79,14 +79,15 @@ const rule: TextlintRuleModule<Options> = (context: TextlintRuleContext, options
 
             // Check for bold list item pattern: - **text**: description
             if (!disableBoldListItems) {
-                const boldListPattern = /^[\s]*[-*+]\s+\*\*([^*]+)\*\*\s*:/;
+                const boldListPattern = /^[\s]*[-*+]\s+\*\*([^*]+)\*\*\s*([:：])/;
                 const boldMatch: RegExpMatchArray | null = text.match(boldListPattern);
                 if (boldMatch) {
                     const matchStart: number = boldMatch.index ?? 0;
                     const matchEnd = matchStart + boldMatch[0].length;
                     const matchRange = [matchStart, matchEnd] as const;
+                    const colon = boldMatch[2];
                     const ruleError = new RuleError(
-                        "リストアイテムで強調（**）とコロン（:）の組み合わせは機械的な印象を与える可能性があります。より自然な表現を検討してください。",
+                        `リストアイテムで強調（**）とコロン（${colon}）の組み合わせは機械的な印象を与える可能性があります。より自然な表現を検討してください。`,
                         {
                             padding: locator.range(matchRange)
                         }

--- a/test/rules/no-ai-list-formatting.test.ts
+++ b/test/rules/no-ai-list-formatting.test.ts
@@ -175,6 +175,17 @@ tester.run("no-ai-list-formatting", noAiListFormatting, {
                     range: [2, 4]
                 }
             ]
+        },
+        // Full-width colon
+        {
+            text: "- **重要情報**：これは重要な項目です",
+            errors: [
+                {
+                    message:
+                        "リストアイテムで強調（**）とコロン（：）の組み合わせは機械的な印象を与える可能性があります。より自然な表現を検討してください。",
+                    range: [0, 9]
+                }
+            ]
         }
     ]
 });

--- a/test/rules/no-ai-list-formatting.test.ts
+++ b/test/rules/no-ai-list-formatting.test.ts
@@ -183,7 +183,7 @@ tester.run("no-ai-list-formatting", noAiListFormatting, {
                 {
                     message:
                         "リストアイテムで強調（**）とコロン（：）の組み合わせは機械的な印象を与える可能性があります。より自然な表現を検討してください。",
-                    range: [0, 9]
+                    range: [0, 11]
                 }
             ]
         }

